app/views/theaters/show.html.erb

```diff
@@ -1,42 +1,73 @@
-<h1 class = "headings"><%=@theater.name%> </h1>
-<p class= "subheadings" style = "margin: auto"><%=@theater.address%>, <%=@theater.city%>, <%=@theater.state%> <%=@theater.zip%></p>
-<br>
-<div class="image_placeholder">
-<%= image_tag "#{@theater.image_url(:thumb)}" %></div>
-
-<% @theater.reviews.each do |review| %>
-  <% @rating_start += review.rating %>
-<% end %>
-
-<% if @rating_start == 0 %>
-  <h3>Be the first to review this theater!</h3>
-<% else %>
-  <h3>RATING : <%= (@rating_start / @theater.reviews.count.to_f).round(1) %> / 5.0 </h3>
-<% end %>
-
-<% if @creator == current_user || (current_user && current_user.admin?) %>
-  <%= link_to "Edit", edit_theater_path(@theater) %> |
-  <%= link_to "Remove #{@theater.name}", @theater, method: :delete%>
-<% end %>
-
-<h5 class="reviews">Reviews</h5>
-  <% @reviews.each do |review| %>
-  <% if review.user.avatar.url.present? %>
-  <div class="profileLogo"> <%= image_tag(review.user.avatar.url)%> <%= review.user.first_name %> Rating: <%= review.rating %>/5 <%= review.body %> <div>
+<head>
+  <link rel="stylesheet" href="show-theater.css">
```

Comment:
This shouldn't be necessary, the asset pipeline should take care of this. What happens when you remove this?